stories/topics/proc-saas-enabling-aws-privatelink.adoc

```diff
@@ -0,0 +1,53 @@
+[id="proc-saas-enabling-aws-privatelink"]
+
+= Enabling {AWSPrivateLink} connectivity
+
+To enable private link connectivity, submit a customer support ticket and the Red{nbsp}Hat team will assist you with next steps. 
+
+[NOTE]
+====
+You must create two support tickets for bi-directional connectivity: one support case for enabling {AWSPrivateLink} connectivity into the control plane and another one for enabling {AWSPrivateLink} connectivity out of the control plane. 
+====
+
+.Procedure 
+
+. Navigate to link:https://access.redhat.com/support/cases/#/case/new/get-support?caseCreate=true&product=Red%20Hat%20Ansible%20Automation%20Platform%20On%20Clouds&version=AWS%20-%20Managed%20Service&seSessionId=db3c1163-09b0-4076-b27f-0ce89eb2b9e7[Customer support]. 
+The fields auto-fill, however confirm: 
+.. The *Account* field displays your customer account
+.. The *Owner* field displays your login
+.. The *Product* field displays *{PlatformName} On Clouds*
+.. The *Version* field displays *AWS - Managed Service*
+. Select *Configuration* for the *What can we help you with?* section.
+. Click btn:[Continue].
+. Use the values in the following templates to copy values into the *Title* and *Describe your problem* sections of the support request.
++
+----
+Title:
+Ansible Automation Platform Service on {AWSPrivateLink} Template
+
+Describe your problem: 
+Enable {AWSPrivateLink} connectivity from control plane to customer's network
```

Comment:
This is rendering as `{AWSPrivateLink}` in the built doc because it's in a code block.
Probably best not to use the attribute in the UI - if AWS changes the name, they'll probably be changing forms etc also